### PR TITLE
526: cyber downloads page

### PR DESCRIPTION
**Pull request by marcelkornblum**

### CODE REVIEW

File: core/migrations/0107_cyber_download.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Generated by Django 4.2 on 2023-09-06 15:37
+
+from django.db import migrations
+
+
+REFERENCE_DATA_MODEL = "CyberDownload"
+OPTIONS = (
+    ("pdf", ".pdf"),
+    ("xls-csv", ".xls / .csv / .xlsm"),
+    ("jpeg-png", ".jpeg / .png"),
```
Comment (SamDudley) [inline core/migrations/0107_cyber_download.py:10]: What do you think to these being snake case? Rational is that makes them valid python names and would match our `TextChoices`.

File: core/models.py [hunk 2]
```diff
@@ -430,6 +434,17 @@ class SharedWith(models.TextChoices):
         null=True,
     )
 
+    cyber_downloads = models.ManyToManyField(
+        "CyberDownload",
+        null=True,
```
Comment (SamDudley) [inline core/models.py:439]: I checked earlier and you don't need `null=True` on `ManyToManyField` it has no effect.

File: core/templates/core/irap_request/steps/downloads.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+    <template data-portal="[data-gds-conditional={{ form.cyber_downloads.html_name }}__other]">
+        {% gds_field form.cyber_downloads_other_info form_group_classes='core-other-field' %}
+    </template>
+
+    {% include './form_footer.html' %}
+</form>
+<script>
+    document.addEventListener("DOMContentLoaded", () => {
+        utils.exclusiveCheckboxOption("{{ form.cyber_downloads.html_name }}", "none_of_the_above");
```
Comment (SamDudley) [inline core/templates/core/irap_request/steps/downloads.html:23]: Should be `"not_applicable"`

**Review by SamDudley — CHANGES_REQUESTED** [review]

File: core/migrations/0107_cyber_download.py (hunk 1 above)
Comment (marcelkornblum) [reply core/migrations/0107_cyber_download.py:10]: yes totally makes sense

File: core/templates/core/irap_request/steps/downloads.html (hunk 3 above)
Comment (marcelkornblum) [reply core/templates/core/irap_request/steps/downloads.html:23]: d'oh!

**Review by SamDudley — APPROVED** [review]

**Pull request merged**